src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const container = document.createElement('div');
+  const containerTitle = document.createElement('h2');
+  const containerDescr = document.createElement('p');
+
+  container.classList.add('notification', type);
```

Comment:
The 'type' parameter should be validated to ensure that it matches one of the expected notification types ('success', 'error', 'warning', etc.). Otherwise, an invalid class could be added to the container.